web/frontend/src/components/slam-2d-render.vue

```diff
@@ -162,15 +284,32 @@ const updateCloud = (pointcloud: Uint8Array) => {
   }
 
   scene.add(points);
-  scene.add(marker);
   scene.add(intersectionPlane);
-};
 
-const updatePose = (newPose: commonApi.Pose) => {
-  const x = newPose.getX();
-  const z = newPose.getZ();
-  marker.position.setX(x);
-  marker.position.setZ(z);
+  const axesHelper2 = new THREE.AxesHelper(5);
+  axesHelper2.position.set(center.x, 0, center.z);
+  axesHelper2.rotateY(-Math.PI / 2);
+  axesHelper2.scale.x = 1e5;
+  axesHelper2.scale.z = 1e5;
+  axesHelper2.renderOrder = 997;
+  axesHelper2.name = 'Axes2';
+  axesHelper1.visible = props.axes;
+
+  // this needs to be updated so it is set to 1m
+  const gridHelper = new THREE.GridHelper(1000, 100, 0xCA_CA_CA, 0xCA_CA_CA);
+  gridHelper.position.set(center.x, 0, center.z);
+  gridHelper.renderOrder = 996;
+  gridHelper.name = 'Grid';
+  gridHelper.visible = props.axes;
+
+  // add objects to scene
+  scene.add(axesHelper1);
```

Comment:
since `scene.add()` accepts variadic arguments maybe:
```ts
scene.add(
  axesHelper1,
  axesHelper2,
  gridHelper,
  points,
  intersectionPlane
)